generator/internal/descriptor_utils.cc

```diff
@@ -409,25 +420,58 @@ ParameterCommentSubstitution substitutions[] = {
     {"\n", "\n  /// "},
 };
 
+// Very long parameters need different formatting.
+auto constexpr kShortParamFormat = "  /// @param %s %s\n";
+auto constexpr kLongParamFormat = R"""(  /// @param %s %s
+  ///     @n
+  ///     For more information, see [%s][%s].
```

Comment:
[See comment in generated code about indentation depth.]

Reply:
Regenerated